applications/solvers/additiveFoam/movingHeatSource/movingHeatSourceModel/movingHeatSourceModel.C

```diff
@@ -98,72 +97,18 @@ void Foam::movingHeatSourceModel::adjustDeltaT(scalar& deltaT)
     }
 }
 
-bool Foam::movingHeatSourceModel::transientExists()
```

Comment:
did you remove the transient melt pool modeling from the code?

Reply:
The transient modeling is not handled in heatSourceModel::updateDimensions() so that it is handled by an individual heat source and not the movingHeatSourceModel class